google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -1208,6 +1208,23 @@ public Notification createNotification(String bucket, Notification notification)
     }
   }
 
+  @Override
+  public Bucket lockRetentionPolicy(Bucket bucket, Map<Option, ?> options) {
+    Span span = startSpan(HttpStorageRpcSpans.SPAN_LOCK_RETENTION_POLICY);
+    Scope scope = tracer.withSpan(span);
+    try {
+      return storage.buckets().lockRetentionPolicy(bucket.getName(), Option.IF_METAGENERATION_MATCH.getLong(options))
```

Comment:
I don't completely understand `IF_METAGENERATION_MATCH`. I assume things work OK if it's null?

Reply:
If it's null it will fail as it's required. Should I pull it into a method parameter instead?

Reply:
Ah, in that case I think it's fine as is.